Cargo.toml

```diff
@@ -14,8 +14,10 @@ members = [
   "iroh-rpc-client",
   "iroh-rpc-types",
   "iroh-share",
+  "iroh-unixfs",
   "iroh-store",
   "iroh-util",
+  "iroh-content",
```

Comment:
are we happy with this name?😅

Reply:
my thoughts here: https://github.com/n0-computer/iroh/pull/566#issuecomment-1339796959

Reply:
@dignifiedquire what do you think about `iroh-content-loader` instead of `iroh-content`. The bulk of the code deals with how to load and interpret content.

Reply:
moving `iroh-content-loader` contents into `iroh-unixfs`